### PR TITLE
[3.x] Better defaults

### PR DESCRIPTION
**Pull request by haringsrob**
- [x] media and files
- [x] Default crops removed from generated models
       @ifox should we maybe change this into a method? GetCrops? Then in config we have "modelCrops" that are defaulting. We can still keep the property, then take them in order: Property there then property, if not, config.
- [x] renderBlocks(false)
- [x] Autoseed default to true
- [x] Disable video URL on media

### CODE REVIEW

File: phpunit.xml [hunk 1]
```diff
@@ -66,10 +66,7 @@
     <env name="DB_CONNECTION" value="sqlite"/>
     <env name="DB_DATABASE" value=":memory:"/>
     <env name="LOG_CHANNEL" value="single"/>
-    <env name="MEDIA_LIBRARY_IMAGE_SERVICE" value="A17\Twill\Services\MediaLibrary\Glide"/>
     <env name="MEDIA_LIBRARY_LOCAL_PATH" value="media-library"/>
-    <env name="MEDIA_LIBRARY_ENDPOINT_TYPE" value="local"/>
-    <env name="FILE_LIBRARY_FILE_SERVICE" value="A17\Twill\Services\MediaLibrary\Glide"/>
```
Comment (haringsrob) [inline phpunit.xml:72]: I am not sure why the file library service was set to glide?

Comment (ifox) [reply phpunit.xml:72]: Looks like an oversight, I don't think this was intentional.

**Comment by haringsrob** [comment]
Screenshot 2022-03-01 at 11 40 33

I changed the output of testbench, also I commented what I think was a leftover, timeout from the test and now they all complete in ~20seconds!

**Comment by ifox** [comment]
WAT?!

I'm not sure if I need more coffee, but I'm not seeing what you commented that changed the tests duration so drastically.

File: tests/integration/TestCase.php [hunk 2]
```diff
@@ -566,9 +566,13 @@ public function copyFiles($files)
...
                 }
 
+                if ($this->files->exists($destination)) {
+                    $this->files->delete($destination);
+                }
+
                 $this->files->copy($source, $destination);
 
-                usleep(1000 * 100); // 100ms
+                /* usleep(1000 * 100); // 100ms */
```
Comment (haringsrob) [inline tests/integration/TestCase.php:575]: @ifox here

Comment (ifox) [reply tests/integration/TestCase.php:575]: 😱

Comment (haringsrob) [reply tests/integration/TestCase.php:575]: I honestly was wondering why the tests took so long before. Guessing it had something to do with testbench.

**Comment by kallefrombosnia** [comment]
Nice changes regarding documentation, very informative. 👍 
A few days ago I have opened small PR #1488 which I have closed since this PR will implement it.
PR was fixing #611 so if possible you can target that issue in this PR and fix it.

**Comment by haringsrob** [comment]
Hey @ifox would appreciate a review on this. Then we can merge this in 3.x

**Comment by haringsrob** [comment]
@ifox, can we remove the configurability of this to always use bigIncrements/bigInteger? Or are there drawbacks to this?

```php
if (!function_exists('twillIncrementsMethod')) {
    /**
     * @return string
     */
    function twillIncrementsMethod()
    {
        return config('twill.migrations_use_big_integers')
            ? 'bigIncrements'
            : 'increments';
    }
}

if (!function_exists('twillIntegerMethod')) {
    /**
     * @return string
     */
    function twillIntegerMethod()
    {
        return config('twill.migrations_use_big_integers')
            ? 'bigInteger'
            : 'integer';
    }
}

```

**Pull request merged**